CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+[0.26.3] - 2017-03-02
```

Comment:
@haikuginger  This change came from merging ENT-191, since it was a dependency.  The final diff shouldn't include this.

But thanks for letting me know whether this change is a major or minor version bump.